workshops/fabric-e2e-rag/workshop.md

```diff
@@ -50,92 +48,82 @@ In this workshop, we'll demonstrate how to develop a context-aware question answ
 
 The architecture of such an application is as shown below:
 
 ![Architecture diagram connecting Azure OpenAI with Azure AI search and Document Intelligence](assets/schema.png)
 
 To get an in-depth understanding of the RAG framework, refer to [this workshop](https://moaw.dev/workshop/gh:azure-samples/azure-openai-rag-workshop/base/docs/)
 
-
 ---
 
-# Environment Setup
+## Environment Setup
+
 To continue with this workshop, you'll need to create a Lakehouse in your Microsoft Fabric workspace and deploy the necessary resources in your Azure account. We'll detail the steps to do this below.
 
-## Create a Lakehouse
+### Create a Lakehouse
+
 To create a new Lakehouse in your Microsoft Fabric workspace, open the `Data Engineering workload` on the bottom left of the workspace and click on `Lakehouse`. Provide the name `rag_workshop` and click `Create`
 
 ![Screenshot of New Lakehouse dialog in Synapse Data Engineering tab](assets/lakehouse.png)
 
 To learn more about Lakehouses in Microsoft Fabric, refer to [this resource](https://learn.microsoft.com/fabric/data-engineering/tutorial-build-lakehouse#create-a-lakehouse?WT.mc_id=data-114676-jndemenge)
 
+### Azure Setup
 
-## Azure Setup
-
-To complete this workshop you'll need an Azure account. If you don't have one, you can create a [free account](https://azure.microsoft.com/free/?WT.mc_id=data-0000-cxa) before you begin. 
-
-<div class="important" data-title="Important">
+To complete this workshop you'll need an Azure account. If you don't have one, you can create a [free account](https://azure.microsoft.com/free/?WT.mc_id=data-0000-cxa) before you begin.
 
+> [!IMPORTANT]
 > Ensure that the subscription you are using has the permissions to create and manage resources.
-</div>
-
 
 Navigate to the [Azure Portal](https://portal.azure.com) and click on `Create a resource` and search for `Azure AI Search`. Click on the `Azure AI Search` resource and then click on `Create`.
 
-![aisearch](assets/aisearch.png)
-
+![Screenshot of Azure AI Search creation wizard in Azure Portal](assets/aisearch.png)
 
 Fill out all the required fields and click `Review + Create` and once the validation is successful, click `Create`.
 
 Similarly, create another resource for the `Azure AI Services` by clicking on `Create a resource` and searching for `Azure AI Services`. Click on the `Azure AI Services` resource and then click on `Create`.
 
-![aiservice](assets/aiservice.png)
+![Screenshot of Azure AI Services creation wizard in Azure Portal](assets/aiservice.png)
 
 Fill out all the required fields, accept the Responsible AI Notice and then click `Review + Create` and once the validation is successful, click `Create`. Ensure that you select the same ***Resource Group*** as the ***Azure AI Search resource***.
 
-## Azure Open AI Set Up
+### Azure OpenAI Set Up
```

Comment:
I changed the headings hierarchy as Markdownlint doesn't like the current hierarchy from an accessibility perspective, but it might actually be correct on the rendered moaw website, so I can change it back.

Reply:
@pamelafox yes I'm testing it ATM but I think it will broke MOAW rendering. By the way, you can test it too. Everything is documented [here](https://moaw.dev/workshop/create-workshop/)

Reply:
It seems to be working 
![image](https://github.com/microsoft/moaw/assets/790974/d2af4821-e3d3-4068-bc70-f62996fbe79a)

Reply:
@pamelafox I spoke too soon. It breaks sub-chapter navigation. Could you revert for now?
@sinedied here is an interesting feedback. TLDR: If we follow markdownlint, we break moaw.

Reply:
Yep, will revert, and will check out the testing doc.

Reply:
That's interesting, it should work though as the first two headings levels are used for the navigation in each page. This is how other workshop are built (example: https://github.com/microsoft/moaw/blob/main/workshops/create-workshop/workshop.md) so there must be something else?